tools/yarn.lock

```diff
@@ -3217,7 +3234,7 @@
   dependencies:
     "@types/node" "*"
 
-"@types/glob@^7.1.1", "@types/glob@^7.1.3":
+"@types/glob@^7.1.1":
```

Comment:
Note, some transitive dependencies still use `glob@7`. Luckily, this shouldn't cause any warnings since expotools are specific to this repository.